docs/Changelog.md

```diff
@@ -11004,6 +11303,61 @@ This version of the operator has been available since version 11 of the default
 <dd>tensor of bool, which should be a scalar.</dd>
 </dl>
 
+### <a name="OneHot-11"></a>**OneHot-11**</a>
```

Comment:
>OneHot-11"></a>**OneHot-11**</a> [](start = 13, length = 32)

We don't need to bump the version of OneHot because the default "axis" value in OneHot-9 is already "-1". It means negative axis is explicitly allowed.